### PR TITLE
IM-62: python 3

### PR DESCRIPTION
**Pull request by mmos7**
**Ticket**:
https://mobilevikings-poland.atlassian.net/browse/IM-62

**Description**:
Ported to Python 3 & Django 2.0

**Actions**:
- [ ] bump version in mv-poland

### CODE REVIEW

File: django_states/log.py [hunk 1]
```diff
@@ -102,6 +103,9 @@ def new_unicode(self):
 
             return ModelBase.__new__(c, class_name, bases, attrs)
 
+        def __str__(self):
```
Comment (meverone) [inline django_states/log.py]: there is no need fo `python_2_unicode_compatible` and `__str__` in metaclass

File: django_states/management/commands/graph_states2.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 logger = logging.getLogger(__name__)
 
 
+@six.python_2_unicode_compatible
```
Comment (meverone) [inline django_states/management/commands/graph_states2.py]: we do not need this

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Dependencies required to develop/test this app.
+
+Django==1.9.12
```
Comment (meverone) [inline requirements.txt]: define it in tox.in as we need it only for tests

File: tox.ini [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
     py27: python2.7
-    py34: python3.4
-deps =
-    six
-    dj16: django==1.6.11
-    dj17: django==1.7.10
-    dj18: django==1.8.4
-    dj19: django==1.9.1
+    py35: python3.5
+deps = -r{toxinidir}/requirements.txt
```
Comment (meverone) [inline tox.ini]:
```
deps =
        dj19: Django>=1.9<1.11
```

Comment (mmos7) [reply tox.ini]: if I do this, I get many import errors when running tox..

Comment (meverone) [reply tox.ini]: **reqiurements.txt**
```
six
```

**tox.init**

```
deps =
    -rrequirements.txt
    dj19: Django>=1.9<1.11
```

Comment (meverone) [reply tox.ini]: also you need to fix `test_proj` and `compat` file as it is deprecated

**Pull request merged**